### ISSUE TITLE
Drivetrain: Add motor controller safety config

### ISSUE THREAD

**Issue by derickson2402** [JHS-Viking-Robotics/FRC-2021#12]
The WPI robotics library contains a safety configuration [detailed here](https://docs.wpilib.org/en/stable/docs/software/labview/resources/adding-safety-features-to-your-robot-code.html) which will automatically stop power to a motor when it has not received a new signal in a specified time interval (the CTRE Phoenix library also contains built in safety timeouts). In Teleop mode, the ```Teleop.vi``` is ran on an occurrance condition whenever a Driver Station (DS) packet is received, which should be 50.0Hz or once approximately every 20ms. Therefore, setting the timeout to 100ms would cause the motors to stop if they miss 5 packets in a row.

The main consideration for this implementation is with Auton mode. ```Autonomous Independent.vi``` is called only once, so any motor update in this VI will need to be used within a loop structure so the signal is updated at least as often as the timeout.

The architecture of the other VI's will need to be double checked as well to ensure the safety timeout will not break anything. One way to debug issues is to set the timeouts to something long like 3 seconds, and if a motor seems to break after about 3 seconds then obviously it is affected by the safety timeout rather than a general bug.

**Comment by derickson2402** [comment]
Duplicate of #14 , keeping open as a reference
